sdk/sdk-e2e/tests/refinanceMakerSparkAnyPair.test.ts

```diff
@@ -0,0 +1,177 @@
+import {
+  Percentage,
+  PositionId,
+  Token,
+  TokenAmount,
+  Position,
+  Address,
+  type Maybe,
+  ChainFamilyMap,
+  newEmptyPositionFromPool,
+  PositionType,
+} from '@summerfi/sdk-common/common'
+
+import { ProtocolName, isLendingPool } from '@summerfi/sdk-common/protocols'
+import { makeSDK, type Chain, type User, Protocol } from '@summerfi/sdk-client'
+import { TokenSymbol } from '@summerfi/sdk-common/common/enums'
+import { IPositionsManager, IRefinanceParameters, Order } from '@summerfi/sdk-common/orders'
+import { ISimulation, SimulationType } from '@summerfi/sdk-common/simulation'
+import { TransactionUtils } from './utils/TransactionUtils'
+
+import { Hex } from 'viem'
+import assert from 'assert'
+import { EmodeType } from '@summerfi/protocol-plugins/plugins/common'
+import { ILKType, MakerPoolId } from '@summerfi/protocol-plugins/plugins/maker'
+import { SparkPoolId, isSparkPoolId } from '@summerfi/protocol-plugins/plugins/spark'
+
+jest.setTimeout(300000)
+
+const SDKAPiUrl = 'https://nkllstfoy8.execute-api.us-east-1.amazonaws.com/api/sdk'
+const TenderlyForkUrl =
+  'https://virtual.mainnet.rpc.tenderly.co/7bebdd10-2878-4830-833d-aba2710f3dc0'
+
+describe.only('Refinance Maker Spark | SDK', () => {
```

Comment:
remove .only

Reply:
And add .skip because those test should not run in CI at the moment